lib/renderers.php

```diff
@@ -7,44 +61,47 @@
  * @param WP_Query $query           The query to render
  * @param integer  $number_of_posts The number of posts to render
  */
-function render_see_also($query, $number_of_posts = 1) {
+function render_see_also($query, $number_of_posts = 1)
```

Comment:
I didn't realise I had changed some of the underlying code and a lot of formatting but I have set my linter to format on save, which would have changed the entire file, is this what you are referring to? I am using the elephant linting plug in but lets compare. 

I'm very confused about this part - changes to the existing codebase would be better to avoid committing and to do in a separate branch, as I am working on my feature branch. I'll get more info when we chat.